### PR TITLE
Dropdown styling

### PR DESCRIPTION
**Pull request by sam-clulow**
Associated Issue:  #5024 

### Summary of Changes

* changed height of dropdown button 
* added display for icons

### Screenshots

icons

### CODE REVIEW

File: assets/images/Svg.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ const svg = {
   webpack: require("./webpack.svg"),
   node: require("./node.svg"),
   express: require("./express.svg"),
+  prettyPrint: require("./prettyPrint.svg"),
```
Comment (jasonLaster) [inline assets/images/Svg.js]: i think this is above :)

File: src/components/shared/Dropdown.css [hunk 2]
```diff
@@ -59,12 +63,14 @@ html[dir="rtl"] .dropdown {
...
   mask-size: 100%;
   background: var(--theme-highlight-blue);
+  display: inline-block;
 }
 
 .dropdown-icon.blackBox {
   mask: url(/images/blackBox.svg) no-repeat;
   mask-size: 100%;
   background: var(--theme-highlight-blue);
+  display: inline-block;
```
Comment (jasonLaster) [inline src/components/shared/Dropdown.css]: Looks good!

**Comment by jasonLaster** [comment]
@lukaszsobek mind taking a look

File: src/components/shared/Dropdown.css [hunk 3]
```diff
@@ -24,6 +24,8 @@ html[dir="rtl"] .dropdown {
   padding: 0px 2px;
   position: relative;
   align-self: center;
+  height: 100%;
+  display: block;
```
Comment (lukaszsobek) [inline src/components/shared/Dropdown.css]: the default display of a div is block, so we might not need this.

File: src/components/shared/Dropdown.css [hunk 4]
```diff
@@ -59,12 +63,14 @@ html[dir="rtl"] .dropdown {
   mask: url(/images/prettyPrint.svg) no-repeat;
   mask-size: 100%;
   background: var(--theme-highlight-blue);
+  display: inline-block;
```
Comment (lukaszsobek) [inline src/components/shared/Dropdown.css]: not sure the inline-block statements have any effect, could you please check on your end?
inline-block

**Pull request merged**